corehq/apps/reports/tasks.py

```diff
@@ -266,26 +248,44 @@ def _format_filename(form_info, question_id, extension, case_id_to_name):
     return filename
 
 
-def _write_attachments_to_file(fpath, use_transfer, num_forms, forms_info, case_id_to_name):
-
-    if not (os.path.isfile(fpath) and use_transfer):  # Don't rebuild the file if it is already there
-        with open(fpath, 'wb') as zfile:
-            with zipfile.ZipFile(zfile, 'w') as multimedia_zipfile:
-                for form_number, form_info in enumerate(forms_info):
-                    form = form_info['form']
-                    for attachment in form_info['attachments']:
-                        filename = _format_filename(
-                            form_info,
-                            attachment['question_id'],
-                            attachment['extension'],
-                            case_id_to_name
-                        )
-                        zip_info = zipfile.ZipInfo(filename, attachment['timestamp'])
-                        multimedia_zipfile.writestr(zip_info, form.get_attachment(
-                            attachment['name']),
-                            zipfile.ZIP_STORED
-                        )
-                    DownloadBase.set_progress(build_form_multimedia_zip, form_number + 1, num_forms)
+def _write_attachments_to_file(fpath, num_forms, forms_info, case_id_to_name):
```

Comment:
I recommend https://github.com/dimagi/commcare-hq/pull/24897/commits/e36bce589de61c9fa0d1efc65933007f76e7b779?w=1 for this commit, since I removed an indentation level